tests/integration/test_charm.py

```diff
@@ -67,14 +67,16 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
     )
     await ops_test.model.add_relation(APP_NAME, POSTGRES)
 
-    async with ops_test.fast_forward():
-        await ops_test.model.wait_for_idle(
-            apps=[APP_NAME, POSTGRES],
-            status="active",
-            raise_on_blocked=True,
-            timeout=1000,
-        )
-        assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
+    await ops_test.model.wait_for_idle(
+        apps=[APP_NAME, POSTGRES],
+        status="active",
+        raise_on_blocked=True,
+        timeout=1000,
+    )
+
+    assert all(
+        unit.workload_status == "active" for unit in ops_test.model.applications[APP_NAME].units
+    )
```

Comment:
Yep, let me remove this check.

I looked through the `wait_for_idle` implementation, I guess it would be better to specify the `wait_for_exact_units` argument in the `wait_for_idle` for this scale-up test case.